conan/cache/conan_reference_layout.py

```diff
@@ -121,6 +122,9 @@ def build(self):
     def package(self):
         return os.path.join(self.base_folder, PACKAGES_FOLDER)
 
+    def post_package(self):
```

Comment:
Lets call it "install" at the moment, not perfect, but ``post_package`` is confusing, seems like the hook and something only after ``package()`` not installation.